napari_aicssegmentation/view/workflow_select_view.py

```diff
@@ -1,63 +1,217 @@
-        self.lbl_select = QLabel("Select a channel")
+        layout = QVBoxLayout()
+        layout.setContentsMargins(0, 0, 0, 0)
+        self.setLayout(layout)
 
-        self.combo_channels = QComboBox()
-        self.combo_channels.currentIndexChanged.connect(self._combo_channels_index_changed)
-        self.combo_workflows = QComboBox()
-        self.combo_workflows.currentIndexChanged.connect(self._combo_workflows_index_changed)
+        # Title
+        workflow_selection_title = QLabel("Workflow selection steps:")
+        workflow_selection_title.setObjectName("workflowSelectionTitle")
 
-        self.btn_back = QPushButton("Back")
-        self.btn_back.clicked.connect(self._btn_back_clicked)
+        # Warning
+        self.load_image_warning = WarningMessage("Open a 3D image in Napari first!")
+        self.load_image_warning.setVisible(False)
 
-        self.btn_next = QPushButton("Next")
-        self.btn_next.clicked.connect(self._btn_next_clicked)
+        # Dropdowns
+        layers_dropdown = self._dropdown_row(1, "Select a 3D Napari image layer", enabled=False)
+        self.combo_layers = layers_dropdown.widget
+        self.combo_layers.currentIndexChanged.connect(self._combo_layers_index_changed)
 
-        self._layout.addWidget(self.lbl_title)
-        self._layout.addWidget(self.lbl_select)
-        self._layout.addWidget(self.combo_channels)
-        self._layout.addWidget(self.combo_workflows)
-        self._layout.addWidget(self.btn_back)
-        self._layout.addWidget(self.btn_next)
+        channels_dropdown = self._dropdown_row(2, "Select a 3D image data channel", enabled=False)
+        self.combo_channels = channels_dropdown.widget
+        layer_channel_selections = self._form_layout([layers_dropdown, channels_dropdown])
+
+        # Add all widgets
+        widgets = [
+            workflow_selection_title,
+            self.load_image_warning,
+            layer_channel_selections,
+        ]
+        for widget in widgets:
+            layout.addWidget(widget)
+        self._add_step_3_layout(layout, enabled=False)
 
     def load_model(self, model: SegmenterModel):
-        self.combo_channels.addItems(model.channel_list)
-        self.combo_workflows.addItems(model.workflows)
+        """
+        Load and display data from model
+        Inputs:
+            model: the model to load
+        """
+        self.update_layers(model.layers, model.selected_layer_name)
+        self.update_channels(model)
+        self._load_workflows(model)
+
+    def update_layers(self, layers: List[str], selected_layer_name: str = None):
+        """
+        Update / repopulate the list of selectable layers
+        Inputs:
+            layers: List of layer names
+            selected_layer_name: (optional) name of the layer to pre-select
+        """
+        if layers is None or len(layers) == 0:
+            self.load_image_warning.setVisible(True)
+            self.combo_layers.setEnabled(False)
```

Comment:
Also need to reset the text shown in the dropdown -- when I deleted a layer, the first dropdown still showed the name of the file I deleted instead of going back to the placeholder text. I think this could be a separate ticket though.